code/src/sixsq/nuvla/server/resources/configuration_nuvla.clj

```diff
@@ -33,13 +34,31 @@ default values.
 (def ^:dynamic *stripe-client-id* nil)
 
 
+(def ^:dynamic *authorized-redirect-urls* nil)
+
+
 (defn throw-stripe-not-configured
   []
   (when-not *stripe-api-key*
     (let [error-msg "server configuration for stripe is missing"]
       (throw (ex-info error-msg (r/map-response error-msg 500))))))
 
 
+(defn is-not-authorized-url?
+  [redirect-url]
+  (not-any? (fn [authorized-redirect] (str/starts-with? redirect-url authorized-redirect)) *authorized-redirect-urls*))
+
+
+(defn throw-is-not-authorised-redirect-url
+  [redirect-url]
+  (if (and redirect-url
+             (seq *authorized-redirect-urls*)
+             (is-not-authorized-url? redirect-url))
+    (let [error-msg (str "server configuration do not authorize following rediect-url: " redirect-url)]
```

Comment:
I suggest extracting the error message as a var and reusing it in the unit test 
https://github.com/nuvla/api-server/pull/549/files#diff-c471681fa6083d06b7e0241fd4a806a813e24debabc66f6a98d53588bb8deb52R174